src/manifests_workflow/input_manifests.py

```diff
@@ -95,76 +94,44 @@ def update(
 
             logging.info(f"Checking {self.name} {branches} branches")
             for branch in branches:
-                c = min_klass.checkout(
-                    path=os.path.join(work_dir.name, self.name.replace(" ", ""), branch),
+                min_component = min_klass.checkout(
```

Comment:
is it just for better readability?
Did not understand the motive behind this change? 
If I understand correctly, instead `OpenSearchDashboards`, this will be `OpenSearch-Dashboard`, right?

Reply:
The `self.name.replace(" ", "")` field is just the folder name after the clone, really does not matter right here.
I just change it to something that is not containing spaces.

Thanks.

Reply:
nit: `min_component_klass`?